app/src/main/res/layout/fragment_subscriptions.xml

```diff
@@ -19,18 +19,31 @@
             app:title="@string/subscriptions_label"
             app:navigationIcon="?homeAsUpIndicator" />
 
-        <TextView
+        <LinearLayout
             android:id="@+id/feeds_filtered_message"
-            android:layout_width="match_parent"
+            android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:paddingBottom="8dp"
-            android:text="@string/filtered_label"
-            android:layout_marginTop="-12dp"
             android:layout_marginLeft="60dp"
             android:layout_marginStart="60dp"
+            android:layout_marginTop="-12dp"
             android:background="?android:attr/selectableItemBackground"
-            android:visibility="gone"
-            tools:visibility="visible" />
+            android:importantForAccessibility="no"
+            android:orientation="horizontal">
+
+            <ImageView
+                android:layout_width="20dp"
+                android:layout_height="20dp"
+                android:src="@drawable/ic_info" />
+
+            <TextView
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginLeft="4dp"
+                android:paddingBottom="8dp"
+                android:text="@string/subscriptions_are_filtered"
+                tools:visibility="visible" />
```

Comment:
This is not needed, it is visible anyway

Reply:
done